client/db_test.go

```diff
@@ -288,3 +290,46 @@ func TestDebugName(t *testing.T) {
 		return nil
 	})
 }
+
+func TestCommonMethods(t *testing.T) {
+	types := []reflect.Type{
+		reflect.TypeOf(&client.DB{}),
+		reflect.TypeOf(&client.Tx{}),
+		reflect.TypeOf(&client.Batch{}),
+		reflect.TypeOf(client.DB{}.B),
+	}
+	blacklist := map[string]bool{
```

Comment:
It's not about space savings IMO, it's the fact that future developers don't have to think about whether there is a difference between a key that is not present and a key that is set to false. We use `map[T]struct{}` pretty consistently.